Lib/test/test_long.py

```diff
@@ -204,7 +204,10 @@ def test_division(self):
         self.check_division(710031681576388032, 26769404391308)
         self.check_division(1933622614268221, 30212853348836)
 
-
+    @support.cpython_only
+    def test_division_with_one(self):
+        for n in range(10000000):
```

Comment:
Well, I don't think that it's worth it to test 10000000 numbers. Just test a bunch of various numbers made of a single digit. Like:

```
(-MASK, -10, -5, 0, 1, 2, 42, MASK)
```